frontend/src/assets/css/scaffolding.scss

```diff
@@ -6,6 +6,7 @@ body {
   display: flex;
   flex-direction: column;
   min-width: 320px;
+  height: 100vh;
```

Comment:
this will cause issues with keyboard on android devices if we will have inputs on a page 🥲  this can be solved with height 100% either here, or maybe if we add it to root
```suggestion
```